templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
Oooh, this is actually because of `"gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '#{project.name}/tools' --strip-components 1 -xf -"`. I haven't even parsed the chocolateyInstall scripts into the directory yet....